contents/docs/self-host/configure/email.md

```diff
@@ -131,9 +130,8 @@ After you have set up your sending configuration, you can continue below to set
     SITE_URL: https://posthog.example.com # this is the URL of your instance
     ```
 
-1. Once you have set these environment variables, restart your server and test sending an email (e.g. request a password reset).
-    
-> **Note:** You will need to restart both your web server and background worker for this to work properly.
+1. Test your integration! Whenever updating email settings, we will attempt to send you a test email so you can assert everything is working as expected. Please note that if you update these settings via environment variables, you need to restart your server and test sending an email (e.g. request a password reset).
```

Comment:
"Whenever updating email settings inside the app" perhaps – just to make it clear when I should expect an automatic email.